web-marketplace/src/components/molecules/CreditsAmount/CreditsAmount.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { Meta, StoryObj } from '@storybook/react';
+import Form from 'web-marketplace/src/components/molecules/Form/Form';
+import { useZodForm } from 'web-marketplace/src/components/molecules/Form/hook/useZodForm';
+import { ChooseCreditsFormSchema } from 'web-marketplace/src/components/organisms/ChooseCreditsForm/ChooseCreditsForm.schema';
+
+import { CreditsAmount } from './CreditsAmount';
+import { PAYMENT_OPTIONS } from './CreditsAmount.types';
+
+export default {
+  title: 'molecules/CreditsAmount',
```

Comment:
any stories within web-marketplace are prefixed with `Registry/` (this is how we called the app previously)
in [one of my recent PRs](https://github.com/regen-network/regen-web/pull/2418), I've updated all web-marketplace stories to use 'Marketplace/'  instead, so you could do that here as well